More news lines (#5786)
endless-sky/endless-sky: data/human/news.txt

			"Do you know how lucky you are, that you can visit a different world every week if you want to? I've been stuck on this one filthy little planet my whole life."
			"If I had enough money, I'd go up to a captain in the spaceport and ask for transport to anywhere else. Nowhere's as bad as this place."
			"Are you hiring new crewmembers? And are you by chance planning to fly to the Paradise Worlds?"

> 			"Whatcha looking at, ya old good-fur-nothing?"

Reviewer: To be honest, this feels like something a sour old man would say to a kid, rather than the reverse (minus the word "old" of course). :)

Author: Perhaps something more like, "Whatcha looking at, ya old geezer?" or "old fart" or "old fossil?"

Reviewer: There ya go, much better!

			"I'll get my act together someday. For now I'm just enjoying life one moment at a time."

> 			"I'd love to get a job, but they all require so much experience, even for those so-called 'entry-level' jobs."

Reviewer: ```suggestion
			"I'd love to get a job, but they all require so much experience, even for those so-called 'entry-level' positions."
```

(otherwise you have two uses of the word "job" in fairly close proximity)

			`"`
		word
			"How is it possible that a man who manages a billion-credit corporation can still be so incompetent or so careless that he routinely misses the toilet bowl when he takes a piss?"

> 			"The family I work for offered to let me live in a room in the attic for free, but I'm sure I want to have that little separation between work and my personal life."

Reviewer: This changes the meaning to be nonsensical

Author: Did you see the next commit I made? I somehow changed that without meaning to, but I fixed it.

			"Sadly, for the last several centuries there have been laws against inflicting bodily harm on people to convince them to pay off their debts."
			"I still think we should be able to charge double the interest on loans to buy a Sparrow, but the regulators won't let us."
			"Do you know how hard it is to force somebody that joined the pirates to pay off their loans?"

> 			"Some people say that we are loan sharks, but I don't understand why. I think that killer whales are way cooler than sharks."

Reviewer: ```suggestion
			"Some people say that we're loan sharks, but I don't understand why. I think that killer whales are way cooler than sharks."
```

(intermixing the use of contractions and expanded forms in the same sentence feels awkward)

			"Read this! 'The Real UGLIEST Starship that Lionheart Don't Want You to Know About!'"
			"Look at my new article! 'The Protector - Actually a SCAM?'"
			"Look at this! 'Five Reasons Why Captains Should Check Spaceports More Often - Number Two Will Make You Cry!'"
			"Read my new article! 'Ten Things the Quarg Are Hiding from Us - Number Seven Will Surprise You!'"

> 			"Check this out! 'Top 20 Things Starship Pilots Don't Want You to Know!' Wait, nevermind."

Reviewer: ```suggestion
			"Check this out! 'Top 20 Things Starship Pilots Don't Want You to Know!' Wait, never mind."
```

("nevermind" isn't a word)

			"Hey, that's a very nice ship you have there. Would be a shame if something happened to it. Do you want to make a donation to the System Safety Fund?"
			"Your credits or your life! Oh you don't have either? Never mind then."
			"Don't count on the Navy to save you if me or any of my friends attack you. Those Navy goons only hang around near the rich planets."

> 			"Did I attack you? Well, I have attacked many people, could you be a bit more specific?"

Reviewer: ```suggestion
			"Did I attack you? Well, I've attacked many people, could you be a bit more specific?"
```

news "human republic reporter in pirate space"
	location
		government "Pirate"
	name
		word
			"Reporter"
	message
		word
			`"`
		word
			"I should never have taken this assignment. Yesterday I interviewed a young pirate and he tried to rob me!"
			"It really isn't so bad here, but I have to make it sound worse than it is to keep my ratings high."

> 			"The political hierarchy out here is way more chaotic than any other planet I've visited. In the past year I've had to report on five coups d'etat!"

Reviewer 2: ```suggestion
			"The political hierarchy out here is way more chaotic than any other planet I've visited. In the past year, I've had to report on five coups d'etat!"
```
Any introductory phrase like this _technically_ needs a comma, but since we're using dialogue, I think it's more appropriate to add commas only when it feels like they're needed. This is a case where I could go either way, just because the sentence feels pretty long without one. Up to you!

Author: I agree that it could go either way, but I've decided to commit your suggestion. Thanks for the input!
